src/js/workshop-comments.js

```diff
@@ -0,0 +1,144 @@
+const commentForm = document.getElementById("comment-form");
+
+const nameDOMElement = document.getElementById("name");
+const commentDOMElement = document.getElementById("comment");
+
+commentForm.addEventListener("submit", (event) => {
+	event.preventDefault();
+
+	const nameDOMElement = document.getElementById("name");
+	const name = nameDOMElement.value.trim();
+	const commentDOMElement = document.getElementById("comment");
+	const comment = commentDOMElement.value.trim();
+
+	const submittedCommentMessage = document.querySelector(".post-failure-message");
+
+	disableFields(true);
+
+	// Hide required message
+	// Note: It's necessary to hide "required" indicators programmatically even though they have been hidden initially by CSS
+	// in the case the same user submits more comments after submitting an initial comment
+	document.querySelectorAll(".required").forEach(element => {
+		applyVisibility(element, false);
+	});
+	// Hide comment submission message
+	// Note: It's necessary to hide succesful/failed submitted comment message indicators programmatically even though they have been hidden initially by CSS
+	// in the case the same user submits more comments after submitting an initial comment
+	document.querySelectorAll(".save-response").forEach(element => {
+		applyVisibility(element, false);
+	});
+
+	// Verify input Fields
+	if ((name === "") & (comment === "")) {
```

Comment:
Line 32-38 can be removed as the individual verification for "name" and "comment" will cover it.

The DOM selector ".required" probably can be removed to as individual selectors of ".required-name" and ".required-comment" cover it too.

Reply:
This case covers when both the `name` field and `comment` field are missing. Without these lines when both fields are missing it will only show the required message for the first field that is empty then hit `return`. (In this case it will only show the required message for the name field when both fields are empty because comes first in the script)

Reply:
I see your point, @TeddTech. A solution is replace `return` in the following verification with setting an error flag such as `isValueMissing = true`. Check the flag after all verification and return if it's `true`.